File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/TimetableService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbTimetable;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface TimetableService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbTimetable> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbTimetable timetable);


	/**
	 * 修改
	 */
	public void update(TbTimetable timetable);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbTimetable findOne(Long id);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbTimetable timetable, int pageNum, int pageSize);
	
}

File: client_seller_web/src/main/webapp/js/service/timetableService.js
//服务层
app.service('timetableService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../timetable/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../timetable/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../timetable/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../timetable/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../timetable/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../timetable/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../timetable/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});